go/genkit/servers.go

```diff
@@ -280,6 +229,189 @@ type listFlowStatesResult struct {
 	ContinuationToken string       `json:"continuationToken"`
 }
 
+// Copyright 2024 Google LLC
```

Comment:
What is this copyright message doing down here?

Reply:
Fixed.